pkg/collector/collector.go

```diff
@@ -79,6 +124,51 @@ func (cc IBCClientsCollector) Collect(ch chan<- prometheus.Metric) {
 				float64(ci.ChainBClientExpiration.Unix()),
 				[]string{(*cc.RPCs)[path.Chain2.ChainName].ChainID, path.Chain2.ClientID, (*cc.RPCs)[path.Chain1.ChainName].ChainID, status}...,
 			)
+
+			stuckPackets, err := ibc.GetChannelInfo(path, cc.RPCs)
+			if err != nil {
+				status = errorStatus
+
+				log.Error(err.Error())
+			}
+
+			for _, sp := range stuckPackets.Channels {
+				ch <- prometheus.MustNewConstMetric(
+					channelStuckPackets,
+					prometheus.GaugeValue,
+					float64(sp.StuckPackets.Total),
+					[]string{
+						sp.Name,
+						(*cc.RPCs)[path.Chain1.ChainName].ChainID,
+						(*cc.RPCs)[path.Chain2.ChainName].ChainID,
+						status,
```

Comment:
When this can have `error` value? I think with current implementation status will be always `success` and no metric when something goes wrong

Reply:
I think it's bad practice to send out metrics if you cannot collect them.
Why would you want to send bad/empty data?

Reply:
With status="error" for a metric it is very easy to build an alert when there is an issue with particular metric. We rely on external services to provide metric data which means it can break at any point in time. No metric no alert in prometheus and it is not very easy to build reliable alerts on metrics which are not always published.

Reply:
There is metrics with `status="error"` 
```
cosmos_ibc_stuck_packets_total{dst_chain_id="cosmoshub-4",dst_channel_id="channel-623",src_chain_id="archway-1",src_channel_id="channel-0",status="error"} 0
```